libs/common/src/assets/i18n/de.json

```diff
@@ -0,0 +1,895 @@
+	"billing.unsubscribe-title": "Es ist nicht vorbei, bis es wirklich vorbei ist ...",
+	"billing.upgrade": "Aktualisierung",
+	"billing.upgrade_now": "Jetzt upgraden",
+	"billing.usage.consumables": "Verbrauchsmaterial",
+	"billing.usage.consumption": "Verbrauch",
+	"billing.usage.interval": "Kosten entsprechend dem Verbrauch von {{from}} bis {{to}}",
+	"billing.usage.over_consume": "Überkonsum",
+	"billing.usage.over_cost": "Zu teuer",
+	"billing.usage.tier_included": "Stufe inklusive",
+	"billing.usage.title": "Kosten für übermäßigen Verbrauch",
+	"billing.vat_id": "Umsatzsteuer-ID",
+	"billing.without_vat": "Preise ohne Mehrwertsteuer",
+	"confirm.delete_body": "Ressource löschen",
+	"confirm.delete_title": "Möchten Sie diese Ressource wirklich löschen?",
+	"dataset.delete": "Beispieldatensatz löschen",
+	"dataset.delete_failed": "Beispieldatensatz konnte nicht gelöscht werden",
+	"dataset.delete_in_progress": "Beispieldatensatz wird gelöscht",
+	"dataset.delete_partially_successful": "Beispieldatensatz teilweise gelöscht (Löschen von {{error}} Ressourcen fehlgeschlagen)",
+	"dataset.delete_successful": "Beispieldatensatz gelöscht",
+	"dataset.import": "Daten importieren",
+	"dataset.import_failed": "Der Import des Beispieldatensatzes ist fehlgeschlagen",
+	"dataset.importing_description": "Dies sollte innerhalb weniger Sekunden erledigt sein ... Fast geschafft! 🙃",
+	"dataset.importing_title": "Datensatz importieren",
+	"dataset.option1.description": "Der schnellste Weg, um loszulegen und die Funktionsweise von Nuclia zu erkunden, ist der Import eines Beispieldatensatzes. Sie können es jederzeit problemlos löschen.",
+	"dataset.option1.title": "Spielen Sie sofort mit einem Beispieldatensatz …",
+	"dataset.option2.description": "Laden Sie einige Ordner und Links hoch oder fügen Sie Ressourcen aus einer CSV-Datei hinzu, damit Sie Nuclia nutzen können. Beachten Sie, dass die Verarbeitung, Kategorisierung und Indexierung der Inhalte einige Zeit in Anspruch nehmen kann.",
+	"dataset.option2.title": "… oder nutzen Sie Ihre eigenen Daten",
+	"dataset.title": "Beginnen Sie mit der Nutzung von Nuclia",
+	"empty.body": "Wir konnten keine relevanten Ergebnisse finden.",
+	"empty.title": "Keine Ergebnisse.",
+	"entities.law": "Rechtliches",
+	"entities.quantity": "Mengen und Maße",
+	"error.creating-kb": "Beim Erstellen der Wissensbox ist ein Fehler aufgetreten",
+	"error.deleting-resource": "Das Löschen der Ressource ist fehlgeschlagen",
+	"error.deleting-resources": "Das Löschen von {{count}} Ressourcen ist fehlgeschlagen",
+	"error.reprocessing-resource": "Die erneute Verarbeitung der Ressource ist fehlgeschlagen",
+	"error.reprocessing-resources": "Die erneute Verarbeitung ist für {{count}} Ressourcen fehlgeschlagen",
+	"farewell.see_again": "Wir hoffen Sie wieder zu sehen.",
+	"farewell.thanks_trust": "Danke für dein Vertrauen!",
```

Comment:
There is some inconsistency, here it is using informal you while usually it is using formal you.